nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/SdkPackageUpdaterHelperTests.cs

```diff
@@ -9,7 +9,7 @@ namespace NuGetUpdater.Core.Test.Utilities
     public class SdkPackageUpdaterHelperTests
     {
         [Fact]
-        public async void DirectoryBuildFilesAreOnlyPulledInFromParentDirectories()
+        public async Task DirectoryBuildFilesAreOnlyPulledInFromParentDirectories()
```

Comment:
Unrelated but async void always spooks me so I thought I'd change it while I was in the area.

Reply:
You bring back memories 😆 ; I'm with you on avoiding `async void`